.travis.yml

```diff
@@ -13,28 +14,15 @@ env:
   global:
     - JEKYLL_ENV=production
     - TZ=US/Pacific # normalize build timestamp
+  matrix:
+    - TASK="./tool/build.sh --check-links"
+    - TASK="./tool/check-code.sh"
+    - TASK="./tool/analyze-and-test-examples.sh"
 
-matrix:
-  include:
-    - env: TASK="bundle exec jekyll build"
-      dart: stable
-    - env: TASK="./tool/check-code.sh"
-      dart: stable
-    - env: TASK="./tool/check-code.sh"
-      dart: dev
-    - env: TASK="./tool/analyze-and-test-examples.sh"
-      dart: stable
-    - env: TASK="./tool/analyze-and-test-examples.sh"
-      dart: dev
-    - env: TASK="./tool/build-and-check-links.sh"
-      dart: stable
-  allow_failures:
-    - env: TASK="./tool/analyze-and-test-examples.sh"
-      dart: dev
-    - env: TASK="./tool/check-code.sh"
-      dart: dev
-    - env: TASK="./tool/build-and-check-links.sh"
-      dart: stable
+# matrix:
+#   allow_failures:
+#     - env: TASK="./tool/analyze-and-test-examples.sh"
+#       dart: dev
```

Comment:
Done. (I usually like to keep an example of the allow_failures section because I tend to forget the syntax so uncommenting is quicker than having to do a look up :).